os-eap64-launch/added/launch/ha.sh

```diff
@@ -71,9 +71,9 @@ function configure_ha() {
 
   JBOSS_HA_ARGS="${JBOSS_HA_ARGS} -Djboss.node.name=${JBOSS_NODE_NAME}"
 
-  if [ -z "${JGROUPS_CLUSTER_PASSWORD}" ]; then
-      log_warning "No password defined for JGroups cluster. AUTH protocol will be disabled. Please define JGROUPS_CLUSTER_PASSWORD."
-      JGROUPS_AUTH="<!--WARNING: No password defined for JGroups cluster. AUTH protocol has been disabled. Please define JGROUPS_CLUSTER_PASSWORD. -->"
+  if [ -z "${JGROUPS_CLUSTER_PASSWORD}" -a "${JGROUPS_ENCRYPT_PROTOCOL}" == "SYM_ENCRYPT"]; then
```

Comment:
ha.sh is executed before jgroups.sh at least for rh-sso... So if user isn't using `JGROUPS_ENCRYPT_PROTOCOL` then this env won't be set at all.  (basically all the user that migrate to new image).

This results in executing `else` block that mis-configures image since `JGROUPS_CLUSTER_PASSWORD` is not set as we already found out.

Reply:
In my opinion, one script shouldn't be aware about other... To keep it simple... I wouldn't touch this script at all...

Reply:
@Crumby Hey Pavel, correct catch WRT to order of execution of `ha.sh` vs `jgroups.sh`.  I wanted the warning about AUTH proto being disabled to be shown only in case `SYM_ENCRYPT` is chosen, and `JGROUPS_CLUSTER_PASSWORD` is undefined.

Updated the PR to reflect this. Also added CTF test to check this, and updated the testing image.

PTAL

Thanks, Jan